test/rest.test.js

```diff
@@ -1578,7 +1578,6 @@ describe('strong-remoting-rest', function() {
           .end(function(err, res) {
             if (err) return done(err);
             var expectedDetail = res.body.error;
-            delete expectedDetail.statusCode;
```

Comment:
hmm...any reason to remove the status code?

Reply:
don't think it should've been deleted to begin with ... tests all still pass. @shimks can confirm

Reply:
There's a breaking change in strong-error-handler 3.0.0 where we _do_ include the status code for each individual errors in the array